src/geometry_aux.cpp

```diff
@@ -387,7 +392,7 @@ prepare_distribcell()
   for (int map = 0; map < target_univ_ids.size(); map++) {
     auto target_univ_id = target_univ_ids[map];
     for (const auto& univ : model::universes) {
-      int32_t offset {0};  // TODO: is this a bug?  It matches F90 implementation.
+      int32_t offset = 0;
```

Comment:
I remember when moving this code to C++ thinking that there might be a bug here if there are two different universes containing the target universe. I'll take a closer look at it and see if there is really a bug there.

Reply:
Ohh, ok that makes more sense now. I read this to mean that it had to do with 0- vs 1-based indexing and it didn't seem relevant. Anyway, let me know what you find.

Reply:
I confirmed that this is not a bug.

It looks weird at first because you'll find a bunch of cells that start with offset 0, but that's totally fine because we add up the offsets from each level of coordinate list.

Reply:
Good news; my removal of that comment is vindicated!